pkg/querier/http.go

```diff
@@ -206,7 +189,7 @@ func (q *QuerierAPI) LogQueryHandler(w http.ResponseWriter, r *http.Request) {
 
 	if r.Header.Get("Accept") == queryrange.ProtobufType {
 		w.Header().Add("Content-Type", queryrange.ProtobufType)
-		err = queryrange.WriteQueryResponseProtobuf(params, result, w)
+		err = queryrange.WriteQueryResponseProtobuf(&params, result, w)
 	} else {
 		w.Header().Add("Content-Type", queryrange.JSONType)
 		err = marshal_legacy.WriteQueryResponseJSON(result, w)
```

Comment:
Should this not be replaced with `WriteResponse`?

Reply:
My comment was lost. Yes, however, it's hard to decide whether to use `marshal_legacy.WriteQueryResponseJSON` or `marshal.WriteQueryResponseJSON` based on the result type. It can only be determined here by the handler itself.